examples/async/composite_agg.py

```diff
@@ -17,19 +17,19 @@
 
 import asyncio
 import os
-from typing import Any, AsyncIterator, Dict, List, Optional, Union
+from typing import Any, AsyncIterator, Dict, Mapping, Sequence
 
 from elasticsearch.helpers import async_bulk
 
-from elasticsearch_dsl import A, Agg, AsyncSearch, Response, async_connections
+from elasticsearch_dsl import Agg, AsyncSearch, Response, aggs, async_connections
 from tests.test_integration.test_data import DATA, GIT_INDEX
 
 
 async def scan_aggs(
     search: AsyncSearch,
-    source_aggs: Union[Dict[str, Agg], List[Dict[str, Agg]]],
+    source_aggs: Sequence[Mapping[str, Agg]],
```

Comment:
The `sources` attribute is declared as a list in the spec, so I'm adapting this example to follow that to avoid typing errors.